### ISSUE TITLE
Navigation: Locales should fallback to the default unless it set

### ISSUE THREAD

**Issue by benfurfie** [statamic/ideas#209]
Right now, you have to create each locales' navigation structure manually. What would be ideal – and far more intuitive – would be that Statamic would fall back to the default navigation for that structure if there isn't one defined locally.

**Comment by tao** [comment]
I agree, I think this would be useful instead of duplicating the navigation X amount of times.
